languages/CRB.groovy

```diff
@@ -0,0 +1,105 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import com.ibm.dbb.metadata.*
+import com.ibm.dbb.dependency.*
+import com.ibm.dbb.build.*
+import com.ibm.dbb.build.report.records.*
+import com.ibm.dbb.build.report.*
+import groovy.transform.*
+
+/*
+The language script is for running the CICS tool (zrb) on CICS definitions in YML (yaml)
```

Comment:
A minor comment.

The _CICS tool_ is a confusing term for me. Instead of 
```
The language script is for running the CICS tool (zrb) on CICS definitions in YML (yaml)
 format ... 
```
can we please rephrase it to:
```
This language script is invoking the 'CICS TS resource builder utility' for the CICS resource definitions defined in the YAML format.

The script processes CICS definition files stored in the git repository with suffix .yml or .yaml to produce a CSD formatted file.

The model file and the application constraints file, required by the CICS TS resource builder, are referenced via the crb.properties files. 

The script expects that the CICS TS resource builder is installed on the build machine in z/OS UNIX. The location of it is also referenced in the properties files.

The output of the process is a CSD formatted file, which is reported in the DBB Build report for further post-build processing such as packaging and deployment into a target environment.

Further information on the CICS TS resource builder utility and can be found at 
 https://www.ibm.com/docs/en/cics-resource-builder/1.0?topic=overview

```

Reply:
Updated as requested

Reply:
Changed as requested